Blog/v1

benjamincerigo opened the pull request:
This is a minimal version of the dvp blog.

You can see the test pages here: https://dvp-landing-static.web.app/

## Implement of the designs
- [blog page](https://app.zeplin.io/project/5e7397e352b634b461fa9cfc/screen/62cbe13349ddf310a2666ee7) 
- [post page](https://app.zeplin.io/project/5e7397e352b634b461fa9cfc/screen/62cbe137fb60a71530750604)
See the comments for the changes that Dan and I agreed on

I have given this a good check but having a check on you mobile and computers would be great. 

Closes: https://github.com/datavaluepeople/dvp-landing-static/pull/59

DBCerigo commented:
Can we get a mouse over affect (change shade, underline, anything) on the post titles 
![image](https://user-images.githubusercontent.com/8318425/211853168-d5687e3f-4b09-4314-b7b2-994dce6e6016.png)

DBCerigo commented:
Can we get a mouse over affect (change shade, underline, anything) on the post titles (on `Test - New Beginnings` in this case)
![image](https://user-images.githubusercontent.com/8318425/211853168-d5687e3f-4b09-4314-b7b2-994dce6e6016.png)

DBCerigo commented:
What's the deal with the weird sized numbers? 
![image](https://user-images.githubusercontent.com/8318425/211854363-b28825fd-f678-45a8-bff2-0676b3d8e5f9.png)

DBCerigo commented on src/components/blog-bio/bio.js:
```diff
@@ -0,0 +1,49 @@
+          />
+          <span className={styles.email}>{email}</span>
+        </div>
+        <p style={{ "margin-bottom": 0 }}>datavaluepeople is a group of data-driven people. Through applied machine learning, building automated systems, advising, and education, we create value for businesses, organizations, and humans. Contact us to find out more about how we can help your organisation or if you are interested in the team.</p>
```
The `Contact us to find out more about how we can help your organisation or if you are interested in the team.` could maybe be better as: `Get in touch us to speak to us about how we could work with your organisation, or if you are interested in joining our team.`

What you think?

DBCerigo replied:
Can we make this sentence bold too? Or that a bit extra? I think it worth it.

benjamincerigo replied:
9f1a533 and e2ff88c

I think the blod is ok. not 100% sure on the visual aspect of it but I think it is good to make that more prominiate. 

One other option is we add a new line to make it more prominate: 
![image](https://user-images.githubusercontent.com/4256007/212260191-8d78c160-00e3-4890-a9a6-343c4cbe3cf0.png)

DBCerigo replied:
There's a typo/mistake in "Get in touch **us** to speak to us...". Since reading it a few time more, I think it would be better if is was: 
```
Drop us an email to speak to us about how we could...
```
Makes it clearer how to get in touch. 
I think I'd prefer doing it bold and not new line.

DBCerigo replied:
This is what is on https://dvp-landing-static.web.app/blog/test/hello-world/ - it doesn't have the bold it doesn't look like. Maybe it needs another build & push? 
![image](https://user-images.githubusercontent.com/8318425/213180598-35522f9c-0b02-4ec1-bb2b-39a6d43a91e4.png)

benjamincerigo replied:
Fixed now and has been deploy to staging2 (dvp-landing-static)

https://github.com/datavaluepeople/dvp-landing-static/pull/66/commits/79c1e7b7dca00ae5cba633de2c01ad6311db16c8

DBCerigo commented:
The author pic kinda borked out after a while: 
![image](https://user-images.githubusercontent.com/8318425/211873555-d4181829-8335-4bf8-8691-52c9beba7396.png)

DBCerigo commented on src/components/discussion-link.js:
```diff
@@ -0,0 +1,33 @@
+        href={link}
+        style={{"text-decoration": "underline"}}
+      >
+        <p>Discuss further on {linkType}</p>
```
I'd make it just `Discuss on...`

benjamincerigo replied:
8ab65a5

DBCerigo commented on docs/track-record-sections.yaml:
```diff
@@ -0,0 +1,9 @@
+# Track Record Sections
```
Think this file has wrong  filename extension

benjamincerigo replied:
477a0f4

DBCerigo commented on src/components/discussion-link.js:
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const DiscussionLink = ({linkType, link}) => {
+  if (link && link != "") {
+    return (
+      <a
```
I think this a great simple solution for this for new. Maybe in future, if these links are working well, then we can upgrade them a bit, like have it `Discuss on: HackerNews, LinkedIn, Reddit, etc.`.

benjamincerigo replied:
Yeah I was hoping to be able to create a map in the meta data but it was not so easy so I did it the simple way first. 

Issue added: https://github.com/datavaluepeople/dvp-landing-static/issues/68

DBCerigo commented on src/components/discussion-link.js:
```diff
@@ -0,0 +1,33 @@
+        href={link}
+        style={{"text-decoration": "underline"}}
+      >
+        <p>Discuss further on {linkType}</p>
```
Do we have tracking by default on if users click these?

benjamincerigo replied:
This should do it: a05d8a9

benjamincerigo replied:
This doesn't work. There is another way of doing it but I am having some problems with it. Is it ok if we remove this commit and deploy it without the analytics and I add this later?

DBCerigo replied:
Yo, just to check, you didn't manage to fix this in the end? So it no analytics on those links?

benjamincerigo replied:
Yeah, I didn't fix it and ai removed the commit, so there are no events being sent to google analytics for clicks on this link. I need to refactor the analytics system because we are currently using out-of-date packages.

DBCerigo commented on src/templates/blog-post.js:
```diff
@@ -0,0 +1,141 @@
+          </div>
+          <div className={styles.navFooter}>
+            <div className={styles.postContainer}>
+              <Link to="/blog">Continue Reading</Link>
```
Reckon `Continue reading` is more natural (than with caps `R`)?

benjamincerigo replied:
ae5ddaa

DBCerigo commented:
author images aren't circles on safari on iOS 
![image](https://user-images.githubusercontent.com/8318425/211925891-2bb8a658-9585-4ede-826e-c88b73b6d22c.png)

DBCerigo commented on src/components/blog-bio/bio.js:
```diff
@@ -0,0 +1,49 @@
+      </div>
+      <div>
+        <p>
+          <ShortBio fullName={fullName}/>
```
Thoughts on adding job name here also?

benjamincerigo replied:
I think go with the design of Gianluca on this one.

DBCerigo commented:
Alright, that's everything I got. 
Looks fuckin sick on mobile (safari - iOS). Looks sick on big desktop screen (firefox). 

Big big ups on it. 

I'm stoked to start getting some writing out there!

benjamincerigo commented:
> What's the deal with the weird sized numbers? ![image](https://user-images.githubusercontent.com/8318425/211854363-b28825fd-f678-45a8-bff2-0676b3d8e5f9.png)

This is the font: https://fonts.google.com/specimen/Raleway?preview.text=123456789&preview.text_type=custom

benjamincerigo commented:
> Can we get a mouse over affect (change shade, underline, anything) on the post titles (on `Test - New Beginnings` in this case) ![image](https://user-images.githubusercontent.com/8318425/211853168-d5687e3f-4b09-4314-b7b2-994dce6e6016.png)

8ab65a5

benjamincerigo commented:
> author images aren't circles on safari on iOS ![image](https://user-images.githubusercontent.com/8318425/211925891-2bb8a658-9585-4ede-826e-c88b73b6d22c.png)

Yep this should do it: 85cd581

benjamincerigo commented:
@DBCerigo deployed on https://dvp-landing-staging.web.app. Ready for rereview. Exciting!

DBCerigo commented:
Nice. Lookin real good. Re-review complete. (@benjamincerigo)

Outstanding items: 
- [ ] Post bio formatting/look https://github.com/datavaluepeople/dvp-landing-static/pull/66#discussion_r1067258021
- [ ] "Discuss on" analytics links clarification https://github.com/datavaluepeople/dvp-landing-static/pull/66#discussion_r1067486552
- [ ] Author pics bork out after refresh https://github.com/datavaluepeople/dvp-landing-static/pull/66#issuecomment-1379220536
    - Steps to recreate: go to https://dvp-landing-static.web.app/blog/test/hello-world/, check profile pic looks ok, refresh the page, check it looks borked. 
    - Confirmed happening on [(Linux, Firefox), (iOS, Safari)]

benjamincerigo commented:
@DBCerigo 

I fixed the bio text and this is deployed on staging2.

I also merged the updates from the other PR and this seems to have solved the image problems. I checked this on firefox and safari with the refresh on the page and it seem to work but if you could have a look that would be great. 

I also noticed a visual problem with the sidebar on our MacBook air. As such I refactored the side bar so that if someone makes the window horizontally small the scroll will be added to the sidebar.

DBCerigo commented:
@benjamincerigo looks facking wicked. See https://github.com/datavaluepeople/dvp-landing-static/pull/66#issuecomment-1387074424 for update on outstanding items. 

Let's get this mofo on staging and then get this liiiiivveee baby
